### PR TITLE
BIOWDL-426: Add new tasks to talon.

### PR DESCRIPTION
**Pull request by JasperBoom**
### Checklist
- [x] Pull request details were added to CHANGELOG.md
- [x] `parameter_meta` for each task is up to date.

### CODE REVIEW

File: talon.wdl [hunk 1]
```diff
@@ -137,12 +137,16 @@ task FilterTalonTranscripts {
         File databaseFile
         String annotationVersion
         String outputPrefix
+        Float maxFracA = 0.5
+        Int minCount = 5
+        Boolean allowGenomic = false
 
-        File? pairingsFile
+        File? datasetsFile
+        Int? minDatasets
```
Comment (DavyCats) [inline talon.wdl:145]: This is not mentioned in the changelog

File: talon.wdl [hunk 2]
```diff
@@ -228,6 +240,57 @@ task GetReadAnnotations {
...
+    runtime {
+        memory: memory
+        time_minutes: timeMinutes
+        docker: dockerImage
+    }
+
+    parameter_meta {
+        # inputs
+        SJinformationFile: {description: "TALON GTF file or database from which to extract exons/introns.", category: "required"}
+        inputFileType: {description: "The file type of SJinformationFile.", category: "required"}
```
Comment (DavyCats) [inline talon.wdl:281]: This input is not required.

File: talon.wdl [hunk 3]
```diff
@@ -228,6 +240,57 @@ task GetReadAnnotations {
...
+        referenceGTF: {description: "GTF reference file (ie GENCODE).", category: "required"}
+        runMode: {description: "Determines whether to include introns or exons in the output.", category: "required"}
```
Comment (DavyCats) [inline talon.wdl:283]: This input is not required.

File: talon.wdl (hunk 1 above)
Comment (JasperBoom) [reply talon.wdl:145]: Updated the CHANGELOG.

File: talon.wdl (hunk 3 above)
Comment (JasperBoom) [reply talon.wdl]: Changed to `common`.

File: talon.wdl (hunk 2 above)
Comment (JasperBoom) [reply talon.wdl:281]: Changed to `common`.

**Pull request merged**